### PR TITLE
Queenie qiu/hw1

### PR DESCRIPTION
**Pull request by QueenieOhYeah**
HW1

### CODE REVIEW

**Comment by zhaohanyun** [comment]
good job

File: MavenProject/project/pom.xml
Comment (flyflyflybird) [inline MavenProject/project/pom.xml:1]: Great work!

**Comment by yuants4213** [comment]
The information is thorough, and the arrangement is easily understandable.

**Comment by oliverXS** [comment]
Great work on this! I appreciate your effort.

**Comment by Frank-Z-Chen** [comment]
LGTM

**Pull request merged**